pkg/debug/checkups/launcher_flags.go

```diff
@@ -0,0 +1,85 @@
+package checkups
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"os"
+	"path/filepath"
+	"runtime"
+
+	"github.com/kolide/launcher/pkg/agent/types"
+	"github.com/peterbourgon/ff/v3"
+)
+
+type launcherFlags struct {
+	k       types.Knapsack
+	status  Status
+	summary string
+}
+
+func (lf *launcherFlags) Name() string {
+	return "Launcher Flags"
+}
+
+func (lf *launcherFlags) Run(_ context.Context, extraFh io.Writer) error {
+	lf.status = Failing
+
+	configFilePath := lf.flagsFilePath()
+	fmt.Fprint(extraFh, fmt.Sprintf("flags file path: %s\n", configFilePath))
+
+	stat, err := os.Stat(configFilePath)
+	if err != nil {
+		lf.summary = fmt.Sprintf("failed to stat %s: %s", configFilePath, err)
+		return nil
+	}
+
+	fmt.Fprint(extraFh, fmt.Sprintf("flags file is %d bytes, and was last modified at %s\n", stat.Size(), stat.ModTime()))
+
+	file, err := os.Open(configFilePath)
+	if err != nil {
+		lf.summary = fmt.Sprintf("failed to open %s: %s", configFilePath, err)
+		return nil
+	}
+
+	fmt.Fprint(extraFh, "Parsed Flags (boolean flags appear as {KEY} TRUE):\n\n")
+
+	if err := ff.PlainParser(file, func(name, value string) error {
```

Comment:
I'm not sure how easy it is to do better -- this might be a good enough for now thing, but this isn't going to be the same parser as [`options.go`](https://github.com/kolide/launcher/blob/main/cmd/launcher/options.go) -- it accepts any options, and won't detect skew

Reply:
Actually, reviewing `parseOptions` I think you can just call it

Reply:
parseOptions lives in the main, cannot reference main outside of main, so I'm looking at moving all the options stuff to it's own pkg

Reply:
moved to own pkg, now using ParseOptions to verify flags